source/_patterns/03-uswds/tag/tag--big/tag--big.md

```diff
@@ -0,0 +1,8 @@
+---
+el: .usa-tag
+title: Tag (Big)
```

Comment:
Same as in #22: we’re using the format “Variant Component” rather than “Component (Variant)” everywhere else in Gus & Gesso.